### DISCUSSION TITLE
Mutation para dar Privilegios/Rangos

### DISCUSSION THREAD

**Discussion by pepepinola** [TeamParches/pastesnip-backend#21] [Ideas]
Tendriamos que tener un intermediario o tercer rango para aplicar la mutation para ser admin, la funcion que tendria esta mutation es dar privilegio a un grupo de usuarios seleccionados por el equipo el cual tiene un rango avanzado

**Comment by larayavrs** [comment]
Opino que está correcta la idea, si un usuario está logeado entonces podria acceder a la mutación que estás diciendo, es decir **cualquiera** siendo usuario puede ser admin, por eso seria bueno agregar un rango mayor a administrador.
